### PR TITLE
Flow SourceLink and Pdb2Pdb versions

### PR DESCRIPTION
**Pull request by tmat**
Automatically propagate the latest SourceLink version that flows into Arcade repo to all repos.

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -71,5 +71,6 @@
     <MicrosoftDiaSymReaderNativeVersion>1.7.0</MicrosoftDiaSymReaderNativeVersion>
     <MicrosoftDotNetMaestroClientVersion>1.1.0-beta.19375.4</MicrosoftDotNetMaestroClientVersion>
     <MicrosoftDotNetSwaggerGeneratorMSBuildVersion>1.0.0-beta.19378.1</MicrosoftDotNetSwaggerGeneratorMSBuildVersion>
+    <MicrosoftSourceLinkVersion>1.0.0-beta2-19367-01</MicrosoftSourceLinkVersion>
```
Comment (jcagme) [inline eng/Versions.props]: Do we need this version to be updated or we want to manually update it whenever needed?

Comment (tmat) [reply eng/Versions.props]: I think we should just flow it automatically. Subscribe Arcade to SourceLink updates.

Comment (jcagme) [reply eng/Versions.props]: Then you also need to add an entry to Version.Details.xml so maestro knows it needs to be updated

Comment (tmat) [reply eng/Versions.props]: Updated

**Pull request merged**